lib/services/ios-debug-service.ts

```diff
@@ -204,12 +205,15 @@ class IOSDebugService implements IDebugService {
 
     private openDebuggingClient(): IFuture<void> {
         return (() => {
-            var cmd = "open -a Safari " + this.getSafariPath().wait();
+            var inspectorPath = this.getInspectorPath().wait();
+            var inspectorApplicationPath = path.join(inspectorPath, "NativeScript Inspector.app");
+            var inspectorSourceLocation = path.join(inspectorPath, "Safari/Main.html");
+            var cmd = util.format("open -a '%s' --args '%s' '%s'", inspectorApplicationPath, inspectorSourceLocation, this.$projectData.projectName);
```

Comment:
You can use [template strings](https://developers.google.com/web/updates/2015/01/ES6-Template-Strings) to simplify the code here.